mantle/platform/api/openstack/api.go

```diff
@@ -594,7 +595,43 @@ func (a *API) UploadImage(name, path, arch, visibility string) (string, error) {
 	return image.ID, nil
 }
 
-func (a *API) DeleteImage(imageID string) error {
+// TODO: We can remove this block when the follow PR lands:
+//       https://github.com/gophercloud/gophercloud/pull/2221
+type ReplaceImageProtected struct {
+	NewProtected bool
+}
+
+func (r ReplaceImageProtected) ToImagePatchMap() map[string]interface{} {
+	return map[string]interface{}{
+		"op":    "replace",
+		"path":  "/protected",
+		"value": r.NewProtected,
+	}
+}
+
+func (a *API) DeleteImage(imageID string, force bool) error {
+	// Detect if the image is protected from deletion. If protected
+	// and force=true then change protection status and delete it.
+	image, err := images.Get(a.imageClient, imageID).Extract()
+	if err != nil {
+		return err
+	}
+	if image.Protected {
+		if force {
+			updateOpts := images.UpdateOpts{
+				ReplaceImageProtected{
+					NewProtected: false,
+				},
+			}
+			image, err = images.Update(a.imageClient, imageID, updateOpts).Extract()
+			if err != nil {
+				return fmt.Errorf("Error removing protection from image: %v", err)
```

Comment:
```suggestion
				return fmt.Errorf("Error removing protection from image %s: %v", imageID, err)
```

If you're feeling fancy, you can use `errors.Wrapf`, though there's still a lot of the codebase that doesn't use it.